riva/clients/nmt/riva_streaming_s2s_client.cc

```diff
@@ -162,12 +171,18 @@ main(int argc, char** argv)
     return 1;
   }
 
+  if (!FLAGS_tts_encoding.empty() && FLAGS_tts_encoding != "pcm" && FLAGS_tts_encoding != "opus") {
```

Comment:
is formatting for the curly brace off here?

Reply:
looks correct, no?
curly brace is at the same line as if, other conditions in the same file also doing similarly (e.g., line 187)

Reply:
Ok. yes. It looked odd due to page render.